pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetParticipantFileDownloadRoute.java

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.ddp.route;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.ActivityInstanceDao;
+import org.broadinstitute.ddp.db.dao.AnswerDao;
+import org.broadinstitute.ddp.db.dto.ActivityInstanceDto;
+import org.broadinstitute.ddp.json.FileDownloadResponse;
+import org.broadinstitute.ddp.json.errors.ApiError;
+import org.broadinstitute.ddp.model.activity.instance.answer.Answer;
+import org.broadinstitute.ddp.service.FileDownloadService;
+import org.broadinstitute.ddp.util.ResponseUtil;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.net.URL;
+import java.util.Optional;
+
+@Slf4j
+@AllArgsConstructor
+public class GetParticipantFileDownloadRoute implements Route {
+    private static FileDownloadService fileDownloadService;
+
+    public GetParticipantFileDownloadRoute(FileDownloadService service) {
+        fileDownloadService = service;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) {
+        String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
```

Comment:
What ensures the request is authorized for this user and that the parameters aren't just guessed?

Reply:
All the User requests that come through DSS server go through UserAuthCheckFilter.java and passed through only if user is Authorized to the study. 
We can discuss if we want to do something more for this API.

Reply:
Right.  We want to make sure they're authorized for the study and that they're either the participant or the proxy.  I don't think our current filter covers that case, does it?

Reply:
I believe thats what UserAuthCheckFilter does 
 canAccess = ddpAuth.canAccessGovernedUsers(requestedUserGuid);
 canAccess = ddpAuth.canAccessStudyDataForUser(requestedUserGuid, study);

Reply:
Added additional Authentication check to make sure user guid and passed instance Guid are Authorized